docs/extensibility/display-widgets.md

```diff
@@ -96,14 +96,19 @@ Columns widgets render the child widgets in two columns.
 
 ### CodeViewer
 
-CodeViewer widgets display values using a read-only code editor. The editor autodetects the language.
+CodeViewer widgets display values using a read-only code editor.
+
+#### Widget-specific parameters
+
+- **language** - used for code highlighting. Editor supports languages handled by [Monaco](https://code.visualstudio.com/docs/languages/overview) and autodetects `yaml` and `json`.
```

Comment:
it automatically sets the language to `yaml` also for jsons so it's misleading

Reply:
Do you suggest no default language? YAML is superset of JSON, so it can understand all JSONs, but not the other way around.